content/en/docs/setup/additional-setup/customize-installation-helm/index.md

```diff
@@ -0,0 +1,161 @@
+
+Using the Helm `post-renderer` capability, you can tweak the installation manifests to meet your requirements easily.
+`Post-rendering` gives the flexibility to manipulate, configure, and/or validate rendered manifests before they are installed by Helm.
+This enables users with advanced configuration needs to use tools like Kustomize to apply configuration changes without the need
+for any additional support from the original chart maintainers.
+
+### Adding a value to an already existing chart
+
+In this example, we will add a `sysctl` value to Istio’s `ingress-gateway` deployment. We are going to:
+
+1. Create a `sysctl` deployment customization patch template.
+1. Apply the patch using Helm `post-rendering`.
+1. Verify that the `sysctl` patch was correctly applied to the pods.
+
+## Create the Kustomization
+
+First, we create a `sysctl` patch file, adding a `securityContext` to the `ingress-gateway` pod with the additional attribute:
+
+{{< text bash >}}
+$ cat > sysctl-ingress-gw-customization.yaml <<EOF
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: istio-ingress
+  namespace: istio-ingress
+spec:
+  template:
+    spec:
+      securityContext:
+          sysctls:
+          - name: net.netfilter.nf_conntrack_tcp_timeout_close_wait
+            value: "10"
+EOF
+{{< /text >}}
+
+The below shell script helps to bridge the gap between Helm `post-renderer` and Kustomize, as the former works with `stdin/stdout`
+and the latter works with files.
+
+{{< text bash >}}
+$ cat > kustomize.sh <<EOF
+#!/bin/sh
+cat > base.yaml
+exec kubectl kustomize # you can also use "kustomize build ." if you have it installed.
+EOF
+$ chmod +x ./kustomize.sh
+{{< /text >}}
+
+Finally, let us create the `kustomization` yaml file, which is the input for `kustomize`
+with the set of resources and associated customization details.
+
+{{< text bash >}}
+$ cat > kustomization.yaml <<EOF
+resources:
+- base.yaml
+patchesStrategicMerge:
+- sysctl-ingress-gw-customization.yaml
+EOF
+{{< /text >}}
+
+## Apply the Kustomization
+
+Now that the Kustomization file is ready, let us use Helm to make sure this gets applied properly.
+
+### Add the Helm repository for Istio
+
+{{< text bash >}}
+$ helm repo add istio https://istio-release.storage.googleapis.com/charts
+$ helm repo update
+{{< /text >}}
+
+### Render and Verify using Helm Template
+
+We can use Helm `post-renderer` to validate rendered manifests before they are installed by Helm
+
+{{< text bash >}}
+$ helm template istio-ingress istio/gateway --namespace istio-ingress --post-renderer ./kustomize.sh | grep -B 4 -A 1 netfilter.nf_conntrack_tcp_timeout_close_wait
+{{< /text >}}
+
+In the output, check for the newly added `sysctl` attribute for `ingress-gateway` pod:
+
+{{< text yaml >}}
+        - mountPath: /etc/envoy
+          name: envoy-config
+      securityContext:
+        sysctls:
+        - name: net.netfilter.nf_conntrack_tcp_timeout_close_wait
+          value: "10"
```

Comment:
Nit as it might vary, but my output here was (we are only told to check the sysctls):
```
          runAsNonRoot: true
          runAsUser: 1337
      securityContext:
        sysctls:
        - name: net.netfilter.nf_conntrack_tcp_timeout_close_wait
          value: "10"
```